CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ cmake_minimum_required (VERSION 3.11)
 # --- Fetch FTXUI --------------------------------------------------------------
 include(FetchContent)
 
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
Could you revert this change?

FTXUI requires C++17, and this is declared as a "public" property, so it should propage to every dependencies:
https://github.com/ArthurSonzogni/FTXUI/blame/b2f66c738626be122ff054504d8335415f018b03/cmake/ftxui_set_options.cmake#L44

Reply:
@ArthurSonzogni I am running MacOS Sonoma on an M1.  When trying to build the example app here, I had to include this change to the `CMakeLists.txt file`.  Without it, I was getting the error:

```ftxui/ftxui-starter-master/build/_deps/ftxui-src/include/ftxui/screen/color.hpp:19:8: error: ISO C++ forbids forward references to 'enum' types```

I am relatively new to building C++ on my mac, is there something else I should do/look at in order to build this sample app?

Reply:
Interesting!
This should work out of the box.
What version of FTXUI are you using?
It would help if there was a CMakeList.txt to reproduce.

Maybe one of your target is not linked correctly to FTXUI? I would expect FTXUI to require [C++17 for everyone using it](https://github.com/ArthurSonzogni/FTXUI/blob/d548a1865876949743af20a1dcbd924b3b0ad8fe/cmake/ftxui_set_options.cmake#L44).